x-pack/plugins/enterprise_search/public/applications/app_search/components/curations/curation/curation.tsx

```diff
@@ -38,20 +40,27 @@ export const Curation: React.FC<Props> = ({ curationsBreadcrumb }) => {
 
   return (
     <>
-      <SetPageChrome trail={[...curationsBreadcrumb, curation.queries.join(', ')]} />
+      <SetPageChrome trail={[...curationsBreadcrumb, queries.join(', ')]} />
       <EuiPageHeader
         pageTitle={MANAGE_CURATION_TITLE}
```

Comment:
I think its worth considering adding the queries to this title

Reply:
As a subtitle, not in the page title, just to clarify? Let me check on this with @daveyholler for sure. For context, here's the standalone UI:

<img width="452" alt="" src="https://user-images.githubusercontent.com/549407/111210245-3ec4d480-858a-11eb-9635-eeb8aa08b34d.png">

Here's Kibana:

<img width="602" alt="" src="https://user-images.githubusercontent.com/549407/111210368-64ea7480-858a-11eb-921d-8a2c1d628fba.png">

I was hoping the Kibana breadcrumbs + the active query switcher would provide enough context that I didn't need to repeat the queries in a subtitle, but definitely fine with adding it if Davey thinks it's needed

Reply:
I was originally going to suggest adding the query to the title here as well, but after giving it some thought I agree that the active query selector and breadcrumbs provide enough context without making us have to worry about how to handle long tail queries as an `<h1>`

Reply:
Thanks Davey! FWIW, I'd be a +1 to revisiting during a polish pass or if we decide we've lost context after coming back to it.